torch/testing/_internal/autograd_function_db.py

```diff
@@ -124,11 +124,11 @@ def forward(x):
         return x
 
     @staticmethod
-    def setup_context(ctx, inputs, outputs):
+    def setup_context(ctx, inputs, output):
         x, = inputs
         ctx.mark_dirty(x)
-        ctx.save_for_backward(outputs)
-        ctx.save_for_forward(outputs)
+        ctx.save_for_backward(output)
+        ctx.save_for_forward(output)
```

Comment:
Should also update the setup_context in autograd_function_db.py to say output instead of outputs while we're at it